superset/config.py

```diff
@@ -445,6 +466,57 @@ class CeleryConfig(object):
 # using flask-compress
 ENABLE_FLASK_COMPRESS = True
 
+# Dashboard v1 deprecation configuration
```

Comment:
@villebro These four need to be removed. Bad rebase with master. My mistake. I will clean it up in a short while

Reply:
@villebro I fixed the bad merge.

Reply:
@mahendra Thanks - I'm still working on this, but due to the sheer size of the PR it's taken longer than I anticipated. Hoping to get comments in before the weekend. Again apologies for the delay.

Reply:
@villebro no worried. Thank you for the review efforts. Appreciate it. :-)